#When installing globally with ebuser:
#  sudo -iu ebuser eb PETSc-3.1-p8-gompi-2016.4.088-download.eb
#I get the following error:
#  Traceback (most recent call last):
#    File "./config/install.py", line 261, in <module>
#      Installer(sys.argv[1:]).run()
#    File "./config/install.py", line 255, in run
#      os.chown(os.path.join(self.rootDir,'conf','petscvariables'),int(os.environ['SUDO_UID']),int(os.e
#  nviron['SUDO_GID']))
#  OSError: [Errno 1] Operation not permitted:
#  '/dev/shm/ebuser/PETSc/3.1-p8/foss-2016.4.088/petsc-3.1-p8/conf/petscvariables'
#
--- install.py	2010-04-08 18:43:47.000000000 +0000
+++ install_new.py	2018-03-22 15:10:17.000000000 +0000
@@ -251,8 +251,8 @@
     fd = file(os.path.join('conf','petscvariables'),'w')
     fd.close()
     # if running as root then change file ownership back to user
-    if os.environ.has_key('SUDO_USER'):
-      os.chown(os.path.join(self.rootDir,'conf','petscvariables'),int(os.environ['SUDO_UID']),int(os.environ['SUDO_GID']))
+    #if os.environ.has_key('SUDO_USER'):
+    #  os.chown(os.path.join(self.rootDir,'conf','petscvariables'),int(os.environ['SUDO_UID']),int(os.environ['SUDO_GID']))
     self.createUninstaller()
     self.outputHelp()
     return
